Analysis/src/EventInfo.cpp

```diff
@@ -239,11 +244,14 @@ double EventInfoBase::GetHT(bool includeHbbJets, bool apply_pt_eta_cut)
     static constexpr double other_jets_max_eta = 4.7;
 
     double ht = 0;
-    const auto& jets = GetJets();
+//    const auto& jets = GetJets();
+    const auto& jets = SelectJets(other_jets_min_pt, other_jets_max_eta);
     for(size_t n = 0; n < jets.size(); ++n) {
         const auto& jet = jets.at(n);
 
-        if(!PassEcalNoiceVetoJets(jet.GetMomentum(), period)) continue;
+//        if(!PassEcalNoiceVetoJets(jet.GetMomentum(), period, event->jets_pu_id.at(n))) continue;
+
+
         if(!includeHbbJets && selected_signal_jets.isSelectedBjet(n)) continue;
```

Comment:
those conditions should be applied when you call `SelectJets` above. Please, review GetHT implementation.